frontend/src/app/home/Pet/Collection.tsx

```diff
@@ -98,15 +107,11 @@ export function Collection() {
       <label htmlFor="owner_field">
         Aptogotchi Minted: {collection?.current_supply}
       </label>
-      <label htmlFor="owner_field">All Holders</label>
+      <label htmlFor="owner_field">Fellow aptogotchis</label>
```

Comment:
what's the proper wording here? first 3 aptogotchis?

Reply:
how about adding **_... and more_** right after those three names? and remove the `[ ]` around the names?

Reply:
updated